servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -110,7 +115,7 @@ public static String getJournalDirectory(String baseDirectory) {
 
   public FileSystemMaster(BlockMaster blockMaster, Journal journal) {
     super(journal,
-        Executors.newFixedThreadPool(2, ThreadFactoryUtils.build("file-system-master-%d", true)));
+        Executors.newFixedThreadPool(2, ThreadFactoryUtils.build("lineage-master-%d", true)));
```

Comment:
Why is this changed to lineage-master if the class name is still FileSystemMaster?